src/app/templates/SeedPhraseInput/SeedLengthSelect.tsx

```diff
@@ -53,7 +55,9 @@ export const SeedLengthSelect: FC<SeedLengthSelectProps> = ({ options, currentOp
     >
       <div className={classNames('flex flex-row justify-around p-2')} onClick={() => setIsOpen(!isOpen)}>
         <span style={{ fontSize: 13 }}>{t('seedInputNumberOfWords', [`${selectedOption}`])}</span>
-        <SelectArrowDownIcon className="ml-1" />
+        <SelectArrowDownIcon
+          className={classNames('ml-1 transition ease-in-out duration-75', isOpen && 'transform rotate-180')}
+        />
       </div>
       <ul className={classNames(!isOpen && 'hidden')}>
         {options.map(option => {
```

Comment:
I would rather make a memoized component for an option and memoize callbacks like https://github.com/madfish-solutions/templewallet-extension/pull/996/files#diff-159324b3d99a8a8f40c08d65956d81911862aefeb72f7bcfce5c140936c129ebR68 there